Sample.java

```diff
@@ -5,3 +5,47 @@
 
 
 // Your code here along with comments explaining your approach
+class MyHashSet {
```

Comment:
Good work with solution but write code description with complexities as well and try the remaining problem